sections/introduction/swi-process.tex

```diff
@@ -0,0 +1,85 @@
+\section{Essence}
+For the process of the project development, we have chosen to work with Essence. 
+Previous semesters we have worked with an agile approach inspired by Scrum, however, this semester we are attempting to apply the Essence approach.
+The basic idea of Essence is to encourage diverse thinking in the team, even though all members of our team shares a similar background as bachelors in Software.
+\\\\
+Essence uses two strategies to support value creation:
+\begin{itemize}
+    \item \textit{A systematic use of diverse viewpoints.} 
+    Values, views and roles are used frequently in Essence. 
+    By using different views and roles to represent problems and solutions, Essence tries to facilitate a range of viewpoints of how a problem needs to be understood and solved.
+    \item \textit{A focus on idea maturation more than idea generation.}
+    Essence applies the concept that ideas develop over time and tries to stimulate the team to evaluate and refine ideas \autocite{Essence}.
+\end{itemize}
+
+\subsection{Four variants of innovation}
+% Overvej om det er nødvendigt
+There are different kinds of innovation which are important within Essence, as the roles considers the different types of innovation. 
```

Comment:
"as the roles considers the different types of innovation" giver ikke rigtig mening. Forstår ikke helt hvad du mener med den sætning